### PR TITLE
add csshake@1.5.0 

### PR DESCRIPTION
**Pull request by Piicksarn**
**git repo url:** https://github.com/elrumordelaluz/csshake/
**Watch 100
  Star 2340
 Fork 403**
@maruilian11,  could you help for me check out the PR, which is referenced to  #6518 ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/csshake/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ],
+  "author": "@elrumordelaluz",
+  "license": "MIT",
+  "homepage": "https://github.com/elrumordelaluz/csshake",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/elrumordelaluz/csshake.git",
+    "basePath": "dist",
+    "files": [
+      "csshake*.css"
```
Comment (PeterDaveHello) [inline ajax/libs/csshake/package.json]: Why not `"csshake*"`?

**Comment by Piicksarn** [comment]
@maruilian11, would you please have a look at the pr?
thank you!!~

**Comment by maruilian11** [comment]
@Piicksarn we prefer npm auto-update, and i think `*.+(js|css)` is better for filemap.

**Comment by Piicksarn** [comment]
@maruilian11, would you mind to have a look again?
Thank you!

**Pull request merged**